app/ts/library/ethereum.ts

```diff
@@ -0,0 +1,32 @@
+import { BigNumber, ethers } from 'ethers'
+import { Result } from 'ethers/lib/utils.js'
+import { TransactionReceipt, TransactionResponse, TransferTransactionResponse } from '../types.js'
+import { ERC20ABI } from './ERC20ABI.js'
+
+export function isTransferTransaction(txResponse: TransactionResponse): txResponse is TransferTransactionResponse {
+	return txResponse.data.toLowerCase().startsWith('0xa9059cbb')
```

Comment:
I don't see this used anywhere but it will only detect `transfer` calls from an EOA. So that doesn't include contract interactions that use `transferFrom` internally.